index.html

```diff
@@ -0,0 +1,631 @@
+                    vast distance from one side of the universe to the other,
+                    and there has not been enough time for scattered patches of hot and cold to mix into an even
+                    temperature.
+                </p>
+            </div>
+            <div class="griditem">
+                <p><strong>DISTANT COFFEE </strong>At a smaller scale, imagine using a telescope to look a mile in one
+                    direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and
+                    how much it has cooled
+                </p>
+            </div>
+            <div class="griditem">
+                <p>
+                    <strong>COFFEE EVERYWHERE </strong> Now turn around and look a mile in the other direction. You see
+                    a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar
+                    cup in every direction, you might want to look for another explanation.
+                </p>
+            </div>
+
+            <div class="griditem1">
+                <p>
+                    <strong>STILL NOT ENOUGH TIME </strong> There has not been enough time to carry coffee cups from
+                    place to place before they get cold. But if all the coffee cups were somehow filled from a single
+                    coffee pot, all at the same time, that might explain their even temperature.</p>
+            </div>
+            <div class="griditem1">
+                <p>
+                    <strong>INFLATION </strong> solves this problem. The theory proposes that, less than a trillionth of
+                    a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in
+                    the violently expanding energy field eventually grew into the large-scale structures of the
+                    universe.
+                </p>
+            </div>
+            <div class="griditem1">
+                <p>
+                    <strong>FLUCTUATION </strong> Astronomers have now detected evidence of these ancient fluctuations
+                    in swirls of polarized light in the cosmic background radiation, which is energy left over from the
+                    early universe. These are gravitational waves predicted by Einstein.
+                </p>
+            </div>
+            <div class="griditem1">
+                <p>
+                    <strong> EXPANSION </strong> Returning to our coffee, imagine a single, central pot expanding faster
+                    than light and cooling to an even temperature as it expands. That is something like inflation. And
+                    the structure of the universe mirrors the froth and foam of the original pot.
+                </p>
+            </div>
+        </div>
+        <hr>
+        <p>By LARRY BUCHANAN and JONATHAN CORUM</p>
+    </section>
```

Comment:
- [ ] Please do not use screenshots of the original page as shortcut to bypass requirements. Please create the 'theory of inflation' section by using grid, flex of float both for images and texts (If you want to use a background image like it is used in the original page, then the view should not be broken in different screen sizes like it is in the original page.)

![image](https://user-images.githubusercontent.com/60184961/90810422-9b7d4b00-e32b-11ea-8035-8c111f331548.png)